prod/docker-compose.yml

```
@@ -0,0 +1,52 @@
+version: '3'
+
+services:
+  
+  php:
+    build: ./php
+    restart: always
+    depends_on: 
+      - postgres
+    networks:
+      - backend
+
+  nginx:
+    image: nginx:1.25.2-alpine
+    ports:
+      - ${NGINX_PORTS:-80:80}
+    environment:
+      SERVER_NAME: "hello.dev"
+    networks:
+      - backend
+    
+  postgres:
+    image: postgres:16.0-alpine3.18
+    restart: always
+    environment:
+      - POSTGRES_DB=${DB_NAME:-root}
+      - POSTGRES_USER=${DB_USER:-root}
+      - POSTGRES_PASSWORD=${DB_PASSWORD:-root}
+    ports:
+      - ${POSTGRES_PORTS:-5432:5432}
+    volumes:
+      - ./db_data:/var/lib/postgresql/data
```

```suggestion
      - db_data:/var/lib/postgresql/data
```
Аналогично с комментарием выше